<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.0.32">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Shawn Wang">
<meta name="dcterms.date" content="2022-08-22">

<title>代谢组部分</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<script src="01.代谢组overview_files/libs/clipboard/clipboard.min.js"></script>
<script src="01.代谢组overview_files/libs/quarto-html/quarto.js"></script>
<script src="01.代谢组overview_files/libs/quarto-html/popper.min.js"></script>
<script src="01.代谢组overview_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="01.代谢组overview_files/libs/quarto-html/anchor.min.js"></script>
<link href="01.代谢组overview_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="01.代谢组overview_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="01.代谢组overview_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="01.代谢组overview_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="01.代谢组overview_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">代谢组部分</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Shawn Wang </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">August 22, 2022</p>
    </div>
  </div>
    
  </div>
  

</header>

<center>
<p><font color="salmon" size="2pt"> 🏡Zhang Lab, Aug 05, 2022 🏡</font></p>
<p><font color="green" size="2pt"> 🔬Multi-omics research center, State Key Laboratory of Crop Stress Adaption and Improvement, HENU, Kaifeng, Henan 🔬</font></p>
</center>
<section id="方法" class="level1">
<h1>方法</h1>
<p><strong>取样方式描述：</strong> 我们在野外进行了黄精块茎的采集，根据经验获取了不同生长年限的材料，粗略分为幼年材料<code>size1-3</code>和成年材料<code>size5-6</code>,<code>size4</code>介于幼年和成年材料之间。</p>
<section id="实验方法" class="level2">
<h2 class="anchored" data-anchor-id="实验方法">实验方法</h2>
<p>赵兵提供；</p>
<section id="metabolites-extraction" class="level3">
<h3 class="anchored" data-anchor-id="metabolites-extraction">Metabolites Extraction</h3>
<p>All collected samples were flash frozen and ground to a fine powder with a mortar and pestle. The powder of each sample was extracted overnight using 80% HPLC grade methanol containing 1 μM chrysin as the internal standard and the ratio of the fresh weight to the volume of extraction solution was kept as 0.1 g/ml. The undissolved sample residues were precipitated by centrifugation at 13000 rpm for 30 min at 4℃. The clear supernatants were loaded into injection vials and ready for UHPLC-MS/MS.</p>
</section>
<section id="high-performance-liquid-chromatography" class="level3">
<h3 class="anchored" data-anchor-id="high-performance-liquid-chromatography">High-Performance Liquid Chromatography</h3>
<p>For UHPLC-MS/MS assay, The vanquish-flex UHPLC system was coupled to Q Exactive Plus mass spectrometry (Thermo Fisher Scientific) for metabolite separation and detection. A Hypersil GOLD column (2.1×100 mm.1.9 um; Thermo Fisher Scientific) was employed for compound separation at 30°C and 5 μl of sample was loaded. The mobile phase A was HPLC grade H2O with 0.1% (v/v) formic acid (Merck, Germany) and phase B was HPLC grade acetonitrile (Merck, Germany). The gradient elution conditions were set as follows: From 0 to 2min, the mobile phase B increases to 10%; From 2 to 10min, the mobile phase B increases to 50%; From 10 to 10.1min， the mobile phase B increases to 80%; From 10.1 to 13 min, the mobile phase B was kept at 80%; From 13 to 14 min the mobile phase B increases to 95%; From 14-18 min the mobile phase B decreased to 10%. The flow rate was 0.3 ml/min. The MS data acquisition was performed by Q Exactive Plus (ThermoFisher Scientific, Rockford, IL) system. In full scan MS/ddMS2 mode, the resolutions of full scan MS and ddMS2 were set at 70000 and 17500, respectively. The automatic gain control (AGC) target and maximum injection time in full scan MS settings were 1e6 and 100 ms, while their values were 2e5 and 50 ms in dd MS2 settings. The TopN (N, the number of top most abundant ions for fragmentation) was set to 8, and collision energy was set to 20%, 40% and 60%. A heated ESI source was used at positive and negative ion mode. the spray voltage was set as 3.5 KV for positive mode and 3.2 KV for negative mode;The capillary temperature and aux gas heater temperature were set as 320 and 350 °C, respectively. Sheath gas and aux gas flow rate were set at 35 and 15 (in arbitrary units), respectively. The S-lens RF level was 50.</p>
</section>
</section>
<section id="数据处理" class="level2">
<h2 class="anchored" data-anchor-id="数据处理">数据处理</h2>
<ol type="1">
<li><p><strong>数据格式转换：</strong> 得到<code>.raw</code>的下机数据后，我们通过<a href="https://hub.docker.com/r/chambm/pwiz-skyline-i-agree-to-the-vendor-licenses">docker版本的ProteoWizard msConvert</a> <span class="citation" data-cites="adusumilliDataConversionProteoWizard2017">Adusumilli and Mallick (<a href="#ref-adusumilliDataConversionProteoWizard2017" role="doc-biblioref">2017</a>)</span> 将原始数据转换为包含MS1信息的<code>.mzXML</code>格式，以及包含MS/MS信息的<code>.mgf</code>格式。</p></li>
<li><p><strong>提峰、数据清洗：</strong> 我们通过<code>tidymass</code> <span class="citation" data-cites="shenTidyMassObjectorientedReproducible2022">Shen et al. (<a href="#ref-shenTidyMassObjectorientedReproducible2022" role="doc-biblioref">2022</a>)</span> 对原始数据进行提峰，数据清洗分为以下几个步骤：</p>
<p>2.1 缺失值检测， 首先对代谢物积累矩阵所有检测到的<code>feature</code>进行缺失值检测，统计。<br>
2.2 噪音去除， 判断一个feature是否为noise的条件为：在所有QC样品中该feature的缺失频率大于20%。<br>
2.3 批次效应检测，去除noise后通过对所有QC样本中feature峰面积统计可以观测到是否有batch effect。<br>
2.4 缺失值填充，使用<code>tidymass</code>的<code>knn</code>算法填充缺失值。<br>
2.5 数据归一化：使用<code>tidymass</code>支持向量机算法对数据进行归一化。 <span class="citation" data-cites="shenNormalizationIntegrationLargescale2016">Shen et al. (<a href="#ref-shenNormalizationIntegrationLargescale2016" role="doc-biblioref">2016</a>)</span>。<br>
2.6 数据整合。</p></li>
<li><p><strong>代谢物注释：</strong> 我们使用以下数据库对检测到的feature进行注释：</p>
<p>3.1 <a href="https://mona.fiehnlab.ucdavis.edu/downloads">MassBank of North America, MoNA</a> MS2;<br>
3.2 <a href="https://massbank.eu/MassBank/">MassBank</a> MS2;<br>
3.3 <a href="https://hmdb.ca/downloads">Human Metabolome Database, HMDB</a> MS2;<br>
3.4 <a href="https://mona.fiehnlab.ucdavis.edu/spectra/browse?query=tags.text%3D%3D%22VF-NPL%22">Vaniya/Fiehn Natural Products Library</a> MS2;<br>
3.5 <a href="https://mona.fiehnlab.ucdavis.edu/spectra/browse?query=tags.text%3D%3D%22RIKEN%20PlaSMA%22">RIKEN PlaSMA</a> MS2;<br>
3.6 Tidymass 内置的RPLC、Fieh_hilic数据库 MS2；<br>
3.7 <a href="http://www.knapsackfamily.com/KNApSAcK/">KNApSAcK</a> MS1;<br>
3.8 <a href="https://plantcyc.org/">PlantCyc</a> MS1;<br>
3.9 <a href="https://www.genome.jp/kegg/">KEGG</a> MS1.<br>
3.10 <a href="http://metdna.zhulab.cn/">MetDNA2</a> 去噪后我们将剩余的feature的MS1及MS2的数据导出，通过<code>MetDNA2</code>对feature进行注释 MetDNA2 <span class="citation" data-cites="shenMetabolicReactionNetworkbased2019">Shen et al. (<a href="#ref-shenMetabolicReactionNetworkbased2019" role="doc-biblioref">2019</a>)</span>；</p></li>
<li><p><strong>注释过滤及整合：</strong> 搜库的时候每个数据库会给出3个匹配度最高的candidates,我们会通过以下方式对注释进行过滤及去冗余：</p>
<p>4.1 对于有MS/MS信息匹配成功的我们保留所有加合离子模式匹配，对于只有MS1匹配的feature，我们只保留<code>[M+H]+</code>以及<code>[M-H]-</code>的结果。<br>
4.2 对于注释到同一代谢物的feature，优先保留有MS/MS信息的feature. 然后根据tidymass给出的匹配度总得分（Total score）选取得分最高的一个feature.<br>
4.3 对于同一个feature注释到不同的代谢物的情况，我们还是优先保留MS/MS匹配的，次优先保留<code>[M+H]+</code>或者<code>[M-H]-</code>模式，最后再根据total score保留。 4.4 对于MetDNA的注释我们没有进行过多的处理，直接将结果整合进来。</p></li>
<li><p><strong>代谢物分类：</strong> 我们根据代谢物的<code>InChiKey</code>信息通过Classyfire <span class="citation" data-cites="djoumboufeunangClassyFireAutomatedChemical2016">Djoumbou Feunang et al. (<a href="#ref-djoumboufeunangClassyFireAutomatedChemical2016" role="doc-biblioref">2016</a>)</span> 数据库将代谢物进行分类，为了更好的可视化分类结果，我们将分类结果中代谢物数量大于等于10个的称为major class，3-9的叫tiny class，小于3个的归类为other class.</p></li>
<li><p><strong>PCA：</strong>我们使用去噪后剩余的feature做PCA分析，首先对标准化后的峰面积做了log10的转换，用R包<a href="http://mirrors.nju.edu.cn/bioconductor/3.15/bioc/html/PCAtools.html">PCAtools</a> 进行PCA计算，用<a href="https://cran.r-project.org/web/packages/scatterplot3d/index.html">scatterplot3D包</a>进行可视化.</p></li>
<li><p><strong>热图：</strong> 我们将最终注释到的化合物通过<a href="https://github.com/jokergoo/ComplexHeatmap">ComplexHeatmap</a> <span class="citation" data-cites="guComplexHeatmapsReveal2016">Gu, Eils, and Schlesner (<a href="#ref-guComplexHeatmapsReveal2016" role="doc-biblioref">2016</a>)</span> 进行了可视化，并且使用<code>column_km=12</code>以及<code>column_km_repeat = 5000</code>参数对代谢物做了5000次的重复的k-means分析,最终将代谢物分了12个cluster. 然后通过计算每个cluster的均值绘制了对应cluster的特征曲线。</p></li>
</ol>
</section>
</section>
<section id="结果" class="level1">
<h1>结果：</h1>
<section id="代谢物提峰过滤注释统计结果" class="level2">
<h2 class="anchored" data-anchor-id="代谢物提峰过滤注释统计结果">代谢物提峰，过滤，注释统计结果：</h2>
<center>
<strong>表1. 数据清洗步骤中feature数量的统计</strong>
</center>
<table class="table">
<thead>
<tr class="header">
<th>Ion</th>
<th>original</th>
<th>remove noise</th>
<th>Annotation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Negative</td>
<td>12316</td>
<td>2156</td>
<td>362</td>
</tr>
<tr class="even">
<td>Positive</td>
<td>16809</td>
<td>3962</td>
<td>414</td>
</tr>
<tr class="odd">
<td>total</td>
<td>29125</td>
<td>6618</td>
<td>776</td>
</tr>
</tbody>
</table>
<center>
<strong>补充表1. 最终776个代谢物积累矩阵</strong>
</center>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>√ Successfully imported: 777 obs. of 8 variables</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>──────────────────────────────────────────────────────────────────────────────────────────────────
       variable_id cluster      Size_1      Size_2      Size_3      Size_4      Size_5      Size_6
──────────────────────────────────────────────────────────────────────────────────────────────────
2   M242T859_POS         8   791735.90   654928.49  1030524.01   616919.80   726557.04   821104.28
3   M409T100_NEG         8  3152627.98  1511003.39  2502632.19  1729881.78  4851488.77  3838031.68
4   M205T945_NEG         8 72172744.41 73235686.18 93915391.24 59677480.56 93061612.83 90236913.75
5   M190T926_POS         8  2796920.05  2028442.93  1595303.99  1554455.97  2069130.69  1831635.13
6   M225T647_POS         8 50187232.11 47584160.29 49994770.89 47019994.41 47568016.41 48163934.97
7   M250T643_POS         8 45644431.08 42795085.45 43049258.28 42828997.81 43807964.37 43359647.61
8   M433T706_POS         8 56238688.66 28708552.91 35308800.95 25002429.18 33411794.57 61669836.76
9   M391T926_POS         8 67546488.22 51855283.23 39894132.08 33184025.66 57274979.46 35274087.84
10  M99T917_POS          8 34956945.71 27022737.79 35498556.80 21912951.87 38284452.64 27842701.93
11  M459T65_NEG          8  9311654.52  4888930.70  5757592.15  3541919.06  8317812.23  6651650.57
12  M693T901_POS         8  1346336.62   545201.62  1104275.50   484409.85   696627.97   471912.69
13  M623T427_NEG         8  1628130.46  1540169.04  3141845.77  1278328.70  2177156.47  2095526.60
14  M455T706_2_POS       8   441454.16   336541.72   665103.07   361602.26   437624.92   502148.21
15  M1035T651_POS        8  3198151.54  1953524.90  2057421.15  2414890.47  2149488.87  2999714.59
──────────────────────────────────────────────────────────────────────────────────────────────────</code></pre>
</div>
</div>
<center>
<strong>补充表2. 代谢物注释</strong>
</center>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>√ Successfully imported: 783 obs. of 16 variables</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
        variable_id                 mz                 rt        Formula                                                   Compound.name    CAS.ID              KEGG.ID            Lab.ID               Adduct                  CE         Total.score       Database Level PubChem.ID                    INCHIKEY
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
2   M1004T543_2_NEG 1004.321744424556  543.20398974609373 C37H66N7O17P3S Palmityl-CoA                                                    1763-10-6 NA                   C00007462         (M-H)-               NA                  0.7676427193513069  KNApSAcK_0.0.1     3  NA        MNBKLUUYKPBKDU-BQVDOTMPNA-N
3   M1007T429_NEG   1007.331749309822  428.86984252929688 C18H32O16      D-(+)-Raffinose                                                 512-69-6  C00492               MassBank_17755    (2M-H)-              Ramp 5-60           0.74353660648588749 MassBank_0.0.2     2  439242    MUPFEKGTMRGPLJ-ZQSKZDJDSA-N
4   M1018T398_POS   1018.478650530821  397.51861572265619 C52H75NO17S    Brevetoxin B1                                                   NA        C20013               C20013            (M+H)+               NA                  0.962052817690777   KEGG_1.0           3  135626479 DRWBCFKGFOPJSK-UHFFFAOYSA-N
5   M101T264_NEG    101.0229074783871  264.47802734375    C4H6O3         Acetoacetic acid                                                541-50-4  NA                   C00007458         (M-H)-               NA                  0.99940229978964124 KNApSAcK_0.0.1     3  NA        WDJHALXBUFZDSR-UHFFFAOYSA-N
6   M101T868_POS    101.07123149964249 867.77178955078125 C4H8N2O        N-Nitrosopyrrolidine                                            930-55-2  C19285               C19285            (M+H)+               NA                  0.99925594244151328 KEGG_1.0           3  124489957 ARQXEQLMMNGFDU-XYFZXANASA-N
7   M102T199_NEG    102.0262321342757  199.30037689208979 C2H5N3O2       Biuret                                                          108-19-0  C06555               C06555            (M-H)-               NA                  0.73887642652619046 KEGG_1.0           3  8784      WBYWAXJHAXSJNI-UHFFFAOYSA-N
8   M102T433_NEG    102.05447506971331 433.10494995117188 C4H9NO2        N,N-Dimethylglycine                                             NA        NA                   MONA_15829        (M-H)-               HCD (NCE 20-30-40%) 0.74910702585330735 MoNA_0.0.1         2  NA        FFDGPVCHZBVARC-UHFFFAOYSA-N
9   M102T452_NEG    102.0545           452                C4H9NO2        (S)-2-aminobutyric acid;Dimethylglycine;gamma-Aminobutyric acid NA        C02356;C01026;C00334 S0148;L0268;S0402 [M-H]-;[M-H]-;[M-H]- NCE30               1                   MetDNA2            2  NA        QWCKQJZIFLGMSD-VKHMYHEASA-N
10  M102T64_1_POS   102.05512087563829 64.298900604248047 C4H9NO3        Homoserine; CE30; UKAUYVFTDYCKQA-VKHMYHEASA-N                   NA        NA                   MONA_19228        (M+H-H2O)+           30                  0.78771460418317318 MoNA_0.0.1         2  NA        UKAUYVFTDYCKQA-VKHMYHEASA-N
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────</code></pre>
</div>
</div>
<center>
<strong>补充表3. 代谢物分类</strong>
</center>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>√ Successfully imported: 632 obs. of 4 variables</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>────────────────────────────────────────────────────────────────────────────────────────────────────
       variable_id                Superclass                       rt                       Subclass
────────────────────────────────────────────────────────────────────────────────────────────────────
2   M182T318_NEG   Acetylides                NA                       NA                            
3   M242T859_POS   Alkaloids and derivatives Tropane alkaloids        NA                            
4   M538T161_NEG   Alkaloids and derivatives Hasubanan alkaloids      NA                            
5   M280T398_NEG   Alkaloids and derivatives Aporphines               NA                            
6   M661T718_POS   Alkaloids and derivatives Macroline alkaloids      NA                            
7   M111T72_NEG    Alkaloids and derivatives Vinca alkaloids          NA                            
8   M918T537_2_NEG Alkaloids and derivatives NA                       NA                            
9   M929T537_NEG   Alkaloids and derivatives NA                       NA                            
10  M532T297_NEG   Alkaloids and derivatives Ergoline and derivatives Lysergic acids and derivatives
────────────────────────────────────────────────────────────────────────────────────────────────────</code></pre>
</div>
</div>
</section>
<section id="结果分析" class="level2">
<h2 class="anchored" data-anchor-id="结果分析">结果分析</h2>
<section id="数据清洗部分" class="level3">
<h3 class="anchored" data-anchor-id="数据清洗部分">数据清洗部分</h3>
<p>经过提峰我们从正谱和负谱共得到29125（pos: 12316, neg:16809）个feature. 经过过滤，我们得到6618（pos: 2156， neg: 3962）个feature，这些feature在至少80%的质控样本中都可以检测到。经过与众多在线数据库比对及MetDNA2的注释，我们最终从正谱筛选出362个从负谱筛选出414个具有较高置信度的，有较为准确注释的化合物进行下游分析。</p>
<p>正谱操作正常，负谱前4个QC存在微弱的批次效应，经过归一化整合，批次效应被去除（Figure S1.）显然这种差异是系统误差（systematic error）。</p>
<p>我们对所有检测到的feature做了PCA分析（Figure S2），经过数据标准化我们发现所有的<code>QC</code>样本都聚在一起，说明系统误差被很好的消除了，负谱的批次效应也被很好的消除。此外我们发现无论在正谱还是负谱成年材料中的<code>size5</code>都明显区别与其他材料，负谱中<code>size6</code>也可以和幼年材料区分开，但似乎过渡材料<code>size4</code>更趋向于幼年材料。当然</p>
<div id="fig-batcheffect" class="quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div id="figS1-Boxplot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="FigS1.batch_effect_boxplot.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Figure S1. Batch effect detection</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div id="figS2-pca" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="FigS2.raw_data_pca.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">FigureS2. Raw data pca</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p></p><figcaption class="figure-caption">Figure 1: batch effect</figcaption><p></p>
</figure>
</div>
</section>
<section id="整体结果分析" class="level3">
<h3 class="anchored" data-anchor-id="整体结果分析">整体结果分析</h3>
<div id="fig-overview" class="quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div id="fig2-overview" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Fig2.Metabo_overview.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Figure 2. overview of metabolomics data</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p></p><figcaption class="figure-caption">Figure 2: overview</figcaption><p></p>
</figure>
</div>

</section>
</section>
</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-adusumilliDataConversionProteoWizard2017" class="csl-entry" role="doc-biblioentry">
Adusumilli, Ravali, and Parag Mallick. 2017. <span>“Data <span>Conversion</span> with <span class="nocase">ProteoWizard msConvert</span>.”</span> <em>Methods in Molecular Biology (Clifton, N.J.)</em> 1550: 339–68. <a href="https://doi.org/10.1007/978-1-4939-6747-6_23">https://doi.org/10.1007/978-1-4939-6747-6_23</a>.
</div>
<div id="ref-djoumboufeunangClassyFireAutomatedChemical2016" class="csl-entry" role="doc-biblioentry">
Djoumbou Feunang, Yannick, Roman Eisner, Craig Knox, Leonid Chepelev, Janna Hastings, Gareth Owen, Eoin Fahy, et al. 2016. <span>“<span>ClassyFire</span>: Automated Chemical Classification with a Comprehensive, Computable Taxonomy.”</span> <em>Journal of Cheminformatics</em> 8 (1): 61. <a href="https://doi.org/10.1186/s13321-016-0174-y">https://doi.org/10.1186/s13321-016-0174-y</a>.
</div>
<div id="ref-guComplexHeatmapsReveal2016" class="csl-entry" role="doc-biblioentry">
Gu, Zuguang, Roland Eils, and Matthias Schlesner. 2016. <span>“Complex Heatmaps Reveal Patterns and Correlations in Multidimensional Genomic Data.”</span> <em>Bioinformatics</em> 32 (18): 2847–49. <a href="https://doi.org/10.1093/bioinformatics/btw313">https://doi.org/10.1093/bioinformatics/btw313</a>.
</div>
<div id="ref-shenNormalizationIntegrationLargescale2016" class="csl-entry" role="doc-biblioentry">
Shen, Xiaotao, Xiaoyun Gong, Yuping Cai, Yuan Guo, Jia Tu, Hao Li, Tao Zhang, Jialin Wang, Fuzhong Xue, and Zheng-Jiang Zhu. 2016. <span>“Normalization and Integration of Large-Scale Metabolomics Data Using Support Vector Regression.”</span> <em>Metabolomics</em> 12 (5): 89. <a href="https://doi.org/10.1007/s11306-016-1026-5">https://doi.org/10.1007/s11306-016-1026-5</a>.
</div>
<div id="ref-shenMetabolicReactionNetworkbased2019" class="csl-entry" role="doc-biblioentry">
Shen, Xiaotao, Ruohong Wang, Xin Xiong, Yandong Yin, Yuping Cai, Zaijun Ma, Nan Liu, and Zheng-Jiang Zhu. 2019. <span>“Metabolic Reaction Network-Based Recursive Metabolite Annotation for Untargeted Metabolomics.”</span> <em>Nature Communications</em> 10 (1): 1516. <a href="https://doi.org/10.1038/s41467-019-09550-x">https://doi.org/10.1038/s41467-019-09550-x</a>.
</div>
<div id="ref-shenTidyMassObjectorientedReproducible2022" class="csl-entry" role="doc-biblioentry">
Shen, Xiaotao, Hong Yan, Chuchu Wang, Peng Gao, Caroline H. Johnson, and Michael P. Snyder. 2022. <span>“<span>TidyMass</span>: <span class="nocase">An Object-oriented Reproducible Analysis Framework</span> for <span>LC-MS Data</span>.”</span> <span>bioRxiv</span>. <a href="https://doi.org/10.1101/2022.03.15.484499">https://doi.org/10.1101/2022.03.15.484499</a>.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->



</body></html>