<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.0.32">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Shawn Wang">
<meta name="dcterms.date" content="2022-10-02">

<title>转录组数据</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<script src="03.2代转录组分析_files/libs/clipboard/clipboard.min.js"></script>
<script src="03.2代转录组分析_files/libs/quarto-html/quarto.js"></script>
<script src="03.2代转录组分析_files/libs/quarto-html/popper.min.js"></script>
<script src="03.2代转录组分析_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="03.2代转录组分析_files/libs/quarto-html/anchor.min.js"></script>
<link href="03.2代转录组分析_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="03.2代转录组分析_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="03.2代转录组分析_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="03.2代转录组分析_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="03.2代转录组分析_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">转录组数据</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Shawn Wang </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">October 2, 2022</p>
    </div>
  </div>
    
  </div>
  

</header>

<center>
<p><font color="salmon" size="2pt"> 🏡Zhang Lab, Aug 05, 2022 🏡</font></p>
<p><font color="green" size="2pt"> 🔬Multi-omics research center, State Key Laboratory of Crop Stress Adaption and Improvement, HENU, Kaifeng, Henan 🔬</font></p>
</center>
<section id="ngs转录组分析" class="level1">
<h1>2 NGS转录组分析</h1>
<section id="方法" class="level2">
<h2 class="anchored" data-anchor-id="方法">2.1 方法</h2>
<ol type="1">
<li><p>以三代全长转录本作为参考序列，将每个样品的clean reads（二代数据）比对回参考序列。该过程我们采用了<code>RSEM</code>软件，<code>RSEM</code>流程中选择使用<code>bowtie2</code>软件，参数为<code>bowtie2</code>默认参数. RSEM对bowtie的比对结果进行统计，进一步得到了每个样品比对到每个基因上的readcount 数目，并对其进行FPKM转换，进而分析基因的表达水平。</p></li>
<li><p>材料间的相关性使用斯皮尔曼相关计算，PCA结果通过PCAtools包完成。</p></li>
<li><p>差异比对软件使用的是<code>DESeq2</code>。</p></li>
<li><p>使用所有差异表达的基因作为基因集合，然后通过<code>mFuzz</code>对这些基因进行分类，我们选择前期和后期差异明显的cluster进行后续分析。</p></li>
<li><p>富集分析使用通过TBtools完成。</p></li>
</ol>
</section>
<section id="结果" class="level2">
<h2 class="anchored" data-anchor-id="结果">3.1 结果</h2>
<section id="global-transcriptomic-changes-of-rhizome-of-p.-sibniricum-between-juvenal-and-adult" class="level3">
<h3 class="anchored" data-anchor-id="global-transcriptomic-changes-of-rhizome-of-p.-sibniricum-between-juvenal-and-adult">3.1.1 Global transcriptomic changes of rhizome of <em>P. sibniricum</em> between juvenal and adult</h3>
<p>为了搞清楚幼年黄精和成年黄精块茎在药效方面的差异，我们以3代全长转录组检测到的转录本为参考，将2代测序的clean reads比对回参考序列，对不同时期的野外黄精块茎样本进行精确的定性和定量。 结果表明平均每个样品产生了78.25 Mb的clean reads, 平均覆盖度为91.55% (Table S9)样品间的相关性表明分属juvenal组的样品和adult组的样品之间相关性较低。由于野外取样的原因，个体差异较为明显，生物学重复间的差异较大，特别是幼年材料，按大小划分整体不容易区分，对于成年材料来说，对于成年材料来说，size5和size6的生物学重复较好，但两者间的相关性非常低，说明差异较大（Figure5 A）,同时，我们对log2转换后的FPKM值计算了欧式距离，并进行了层级聚类，距离方法为<code>complete</code>,同时也做了PCA分析，结果显示不同年份的材料较好的聚集在一起(Figure S5 A,B)，且Size5和其他材料差异较大。 为了研究不同年份黄精之间的差异，我们对他们做了成对的差异分析，与材料相关性分析结果一致，size5和size6与其他组合差异基因数量最多，同时size5和size6之间也存在较大差异。综上，我们发现在转录水平幼年黄精和成年黄精间确实存在明显差异，而两个成年黄精材料间也存在较为明显的差异，尽管幼年黄精间可能由于个体差异的关系生物学重复不太理想，造成重复间相关性较低，但是在压缩数据变异度后（对FPKM取log）不同大小的黄精块茎材料间的差异还是被凸显出来，所以通过经验判断黄精年份的方法确实存在一定问题，但是在转录水平确实体现出一定的趋势，同时在如此大的个体差异存在的情况下，幼年黄精和成年黄精间显著的差异表明黄精在生长发育到一定阶段后，块茎内正在进行的生物学过程及代谢途径确实发生了显著的改变。从幼年黄精和成年黄精材料的转录及代谢数据中去规避个体差异造成的影响从而挖掘真实的年份积累造成的差异成为本研究关注的重点。</p>
<div id="fig5.DET_overview" class="quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div id="Def" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Fig5.DET_overview.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Figure5.DET.overview</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="FigS5.NGS_Trans_Overview.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">FigureS5.DET.overview2</figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
</section>
</section>
<section id="time-course-transcriptome-analysis" class="level2">
<h2 class="anchored" data-anchor-id="time-course-transcriptome-analysis">3.1.2 Time course transcriptome analysis</h2>
<p>为了更好的了解这些DETs在不同年份之间的表达变化，我们通过mFuzz对DETs做了时序分析，这些DETs被分成了12个cluster。这些DETs表现出多样的趋势，在这些expression pattern中，我们更加关注在Adult中特异表达的DETs，其中cluster2,8,9表现出明显的在材料size5中高表达，而在其他时期相对较低（Figure 5 A）。通过后续的GO富集分析我们发现，cluster2的DETs更多的影响的是非编码RNA代谢通路<code>ncRNA metabolic process</code>,泛素依赖性蛋白质分解代谢过程<code>ubiquitin−dependent protein catabolic process</code>(Figure S6 A)。cluster8与cluster2的功能类似，同样也是影响 <code>RNA metabolic process</code>，<code>protein modification process</code>.然而 cluster9则显著富集在碳水化合物代谢，多糖代谢过程等通路（Figure 6 B）从表达模式上看，Size5中，除了在nuclear metabolic和protein modification这些pathway上调之外，多糖代谢，特别是多糖合成代谢也上调了。值得注意的是cluster 9中除了Size5外，这些基因在Szie1中表达量同样较高，而同属Adult的size6特意表达的基因在cluster11，GO富集分析的结果显示这些DETs更多的参与到了氧化还原，有机酸的生物合成代谢通路、苯丙烷的代谢通路中（Figure 6 C）从cellular component trem的富集结果看，这些DETs与质膜相关，参与了植物细胞壁的形成,从KEGG富集分析的结果看α-亚麻酸代谢也显著被富集(Figure S 6C)。此外我们还发现在幼年黄精材料中特异性表达的一些cluster，比如cluster3在Size3阶段特意高表达，cluster4在size4阶段特意高表达，cluster10在size2阶段高表达，cluster12在size1阶段高表达，cluster5在size1-2阶段高表达。综上，我们通过时序分析挖掘出老年黄精和幼年黄精在整体表达模式上的异同，结果表明，相对于幼年黄精，成年黄精在<code>RNA metabolic process</code>，<code>protein modification process</code>，<code>obsolete oxidation-reduction progress</code>,<code>phenylpropanoid metabolic process</code>较为活跃，基于GO富集分析的结果，我们发现似乎Size1和Size5多糖合成代谢要比其他时期更强。</p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>√ Successfully imported: 29 obs. of 7 variables</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
                      Gene_ID Category        KEGG_entry                  KOG_class_annotation                                                                                      Uniprot Annotation                                                                                                                                     NR Annotation                                                                                                   NT Annotation
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
1  HJ_1-10k_transcript/267397     glgB K00700|GBE1, glgB --                                    1,4-alpha-glucan-branching enzyme 1, chloroplastic/amyloplastic-like [Asparagus officinalis]            PREDICTED: Asparagus officinalis 1,4-alpha-glucan-branching enzyme 1, chloroplastic/amyloplastic-like (LOC109821070), mRNA                        1,4-alpha-glucan-branching enzyme, chloroplastic/amyloplastic OS=Oryza sativa subsp. japonica GN=SBE1 PE=1 SV=2
2  HJ_1-10k_transcript/61446      glgB K00700|GBE1, glgB Carbohydrate transport and metabolism 1,4-alpha-glucan-branching enzyme 1, chloroplastic/amyloplastic-like [Asparagus officinalis]            PREDICTED: Asparagus officinalis 1,4-alpha-glucan-branching enzyme 1, chloroplastic/amyloplastic-like (LOC109821070), mRNA                        1,4-alpha-glucan-branching enzyme, chloroplastic/amyloplastic OS=Oryza sativa subsp. japonica GN=SBE1 PE=1 SV=2
3  HJ_1-10k_transcript/162055     glgB K00688|PYG, glgP  --                                    alpha-1,4 glucan phosphorylase L isozyme, chloroplastic/amyloplastic isoform X2 [Asparagus officinalis]                                                                                                                                                   Alpha-1,4 glucan phosphorylase L-2 isozyme, chloroplastic/amyloplastic OS=Solanum tuberosum GN=STP-1 PE=1 SV=1 
4  HJ_1-10k_transcript/310449     glgB K00688|PYG, glgP  --                                    alpha-1,4 glucan phosphorylase L isozyme, chloroplastic/amyloplastic-like isoform X1 [Glycine soja]                                                                                                                                                       Alpha-1,4 glucan phosphorylase L-2 isozyme, chloroplastic/amyloplastic OS=Solanum tuberosum GN=STP-1 PE=1 SV=1 
5  HJ_1-10k_transcript/14006      abfA K01209|abfA       --                                    alpha-L-arabinofuranosidase 1-like [Asparagus officinalis]                                              PREDICTED: Asparagus officinalis alpha-L-arabinofuranosidase 1-like (LOC109841490), mRNA                                                          Alpha-L-arabinofuranosidase 1 OS=Arabidopsis thaliana GN=ASD1 PE=1 SV=1                                        
6  HJ_1-10k_transcript/13135      CALS K11000|CALS       --                                    callose synthase 3-like [Asparagus officinalis]                                                         PREDICTED: Asparagus officinalis callose synthase 3-like (LOC109839031), mRNA                                                                     Callose synthase 3 OS=Arabidopsis thaliana GN=CALS3 PE=2 SV=3                                                  
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────</code></pre>
</div>
</div>
<div id="fig6.mFuzz" class="quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="Fig6.mFuzz.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Figure6.mFuzz</figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
</section>
<section id="黄精多糖合成代谢通路分析" class="level2">
<h2 class="anchored" data-anchor-id="黄精多糖合成代谢通路分析">3.1.3 黄精多糖合成代谢通路分析。</h2>
<p>为了更好的了解黄精多糖的代谢通路在幼年黄精和成年黄精之间的差异，我们基于我们三代全长转录组注释信息及 <span class="citation" data-cites="wangNovoAssemblyAnalysis2017">Wang et al. (<a href="#ref-wangNovoAssemblyAnalysis2017" role="doc-biblioref">2017</a>)</span> ，<span class="citation" data-cites="fengTranscriptomeAnalysisDifferent2022">Feng et al. (<a href="#ref-fengTranscriptomeAnalysisDifferent2022" role="doc-biblioref">2022</a>)</span> 总结的proposed pathways for polysaccharide biosynthesis in <em>polygonatum sibiricum</em>. 对幼年黄精和成年黄精之间的黄精多糖代谢途径探索，首先我们基于注释数据库对基因进行了筛选，我们保留了NR, NT以及swissport数据库比对结果中至少有两个数据库E-value &lt; 1E-100. 其次使用至少有两个sample中FPKM &gt; 1的条件对结果进一步筛选，最终得到128个高可信的PSP-related genes (Table S10),其中有118个属于DET. 我们将注释到同一基因的isoform的FPKM值合并，然后计算了Juvenal和Adult样品间这些基因的表达量变化得到以下结果。</p>
<p>根据该推断出来的黄精多糖合成通路，我们发现植物在经过光合作用合成蔗糖后，蔗糖通过<code>β-fructofuranosidase (*sacA*)</code>水解成<code>Fructose</code>和<code>Glucose 6-phosphate (Glc-6P)</code>,果糖在果糖激酶<code>fructokinase (*scrK*)</code>及<code>hexokinase (*HK*)</code>的作用下被磷酸化为6磷酸果糖 <code>Fructose-6P (Fru-6P)</code> ，同时，G6P也会在葡萄糖6磷酸异构酶<code>glucose-6-phosphate isomerase (GPI)</code>的作用下转变为F6P。一部分的果糖会转变为甘露糖(Mannose)，甘露糖在<em>HK</em>的作用下被磷酸化为6磷酸甘露糖<code>Mannose-6P (Man-6P)</code>，此外，F6P也会在甘露糖 6-磷酸异构酶<code>mannose-6-phosphate isomerase (*MPI*)</code> 的作用下转变为M6P。而M6P会在磷酸腺苷变位酶<code>phosphomannomutase (*PMM*)</code>的作用下变为1磷酸甘露糖(<code>Mannose-1P</code>)，然后在甘露糖-1-磷酸鸟苷转移酶<code>mannose-1-phosphate guanylyltransferase (*GMPP*)</code>的作用下转变为<code>GDP-Mannose</code>而<code>GDP-Mannose</code>一方面通过GDP甘露糖4，,6脱水酶<code>GDPmannose 4,6-dehydratase (*GMDS*)</code>脱去一分子水变为<code>GDP-4-keto-6-deoxy-D-mannose</code>，然后在GDP果糖合酶<code>GDP-L-fucose synthase (*TSTA3*)</code>的作用下形成GDP果糖<code>GDP-Fuc</code>，最终聚合成多糖,另一方面通过多步反应最终合成甘露聚糖。在该途径中，我们发现<em>sacA</em>、<em>scrK</em>及<em>MPI</em>在成年黄精中表达量明显高于幼年黄精，说明在成年黄精中，由蔗糖向葡萄糖及果糖转化过程明显增强，同时果糖被磷酸化为F6P的过程同样也增强，F6P异构化转变为甘露糖的过程也增强，所以似乎成年黄精在合成<code>GDP-Monnose</code>被加强了。同时我们发现甘露糖磷酸化的过程在成年黄精中有所减弱，同时转变为<code>1,4-β-Mannan</code>的能力也减弱，同时<code>GDP-Monnose</code>转变为<code>GDP-Fuctose</code>的过程由于<em>TSTA3</em>的显著下调也减弱了。由此我们可以推断出成年黄精合成甘露聚糖的能力要优于幼年黄精。</p>
<p>另一条通路为G6P在磷酸葡萄糖变位酶<code>phosphoglucomutase (*pgm*)</code>的作用下转化为G1P，G1P一方面通过<code>Glucose-1-phosphate adenylyltransferase (*glgc*)</code> 转化为<code>ADP-Glc</code>，或者通过多步反应生成纤维二糖(cellobiose),纤维二糖在<code>beta-glucosidase (*Bglu*)</code> 的作用下转换为葡萄糖。在这两条途径中<code>glgc</code>在成年黄精中显著上调，说明更多的G1P直接转化成ADP-Glc。此外我们在代谢组数据中发现纤维二糖在成年黄精中积累量要显著高于幼年黄精，我们推测成年黄精中G1P在合成纤维素(cellulose)后会发生水解生成纤维素糊精，纤维二糖等，而<code>Bglu</code>的下调导致这些纤维素水解产物不能快速的转化为葡萄糖从而在成年黄精中积累。G1P还可以在UTP——葡萄糖-1-磷酸尿苷酰转移酶<code>UTP--glucose-1-phosphate uridylyltransferase (*galU*)</code>的作用下变为UDP-Glc，然后在<code>UDP-glucose 4,6-dehydratase (*RHM*)</code>的催化下变为<code>UDP-4-keto-6-deoxy-D-Glc</code> 在经过多不反应生成UDP鼠李糖<code>UDP-Rha</code>,最终形成多糖。此外UDP-Glc在UDP葡萄糖4-差向异构酶<code>UDP-glucose 4-epimerase (GALE)</code>的作用下与UDP半乳糖<code>UDP-Gal</code>相互转换,<code>UDP-Gal</code>分解为乳糖和半乳糖后在β半乳糖苷酶的作用下合成<code>D-Gal</code>，然后多步反应生成<code>α-D-G1P</code>。另一方面UDP-Glc在<code>UDP-glucose 6-dehydrogenase</code>作用下生成UDP-葡萄糖醛酸<code>UDP-GlcA</code>,<code>UDP-GlcA</code>在差向异构酶<code>UDP-glucuronate 4-epimerase (*UGE*)</code>的作用下与<code>UDP-GalA</code>互相转化。UDP-GlcA会在<code>UDP-apiose/xylose synthase (*AXS*)</code>和<code>UDP-glucuronate decarboxylase (*UXS*)</code>的作用下生成<code>UDP-D-Xylose</code>和UDP-D-芹菜糖<code>UDP-D-Apiose</code>，<code>UDP-D-Xylose</code>会在<code>UDP-arabinose 4-epimerase (*UXE*)</code> 作用下形成<code>UDP-D-Aarabinose</code>。在这个过程中我们发现<em>UGDH</em>，<em>GAE</em>在成年黄精中显著高表达，说明在在UDP-Glc和UDP-Gal之间通过转变为对应的UDP-糖醛酸形式进行转变的活动更加活跃。和<em>AXS</em>和<em>UXS</em>的上调也说明在成年黄精中UDP-Glc转变为其他UDP-糖的活动较为活跃。</p>
<p>综上，我们发现在成年黄精中蔗糖分解途径更加活跃，产生黄精多糖的底物相应的会增多，而多糖种类中，我们发现成年黄精更倾向于积累甘露聚糖，且从UDP-Glc转变为其他类型UDP-糖的能力也要由于幼年黄精。</p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>√ Successfully imported: 130 obs. of 20 variables</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
                       GeneID category                                                                                                                     GO_Anno KEGG_entry         ID                 E_value       Identity Score                                                      Annotation             ID E_value         Identity Score                                                                                      Annotation                   ID                 E_value       Identity Score                                                                     Annotation DET set
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
3  HJ_1-10k_transcript/257150      AXS GO:0003824: catalytic activity (molecular_function); GO:0050662: coenzyme binding (molecular_function);                     K12449|AXS AMM04380.1 6.8999999999999999E-209 347/374(92.78)   732 UDP-D-apiose/UDP-D-xylose synthase [Albuca bracteata]           XM_020421333.1       0 1002/1151(87.05)  1284                                                                                                 sp|Q9SGE0|AXS2_ARATH 1.4000000000000001E-202 332/373(89.01)   706 UDP-D-apiose/UDP-D-xylose synthase 2 OS=Arabidopsis thaliana GN=AXS2 PE=2 SV=1    TRUE
4  HJ_1-10k_transcript/209937      AXS GO:0003824: catalytic activity (molecular_function); GO:0050662: coenzyme binding (molecular_function);                     K12449|AXS AMM04380.1 3.9000000000000001E-217 361/391(92.33)   760 UDP-D-apiose/UDP-D-xylose synthase [Albuca bracteata]           XM_020421333.1       0 1022/1174(87.05)  1310                                                                                                 sp|Q9SGE0|AXS2_ARATH 5.8000000000000005E-209 343/392(87.50)   728 UDP-D-apiose/UDP-D-xylose synthase 2 OS=Arabidopsis thaliana GN=AXS2 PE=2 SV=1    TRUE
5  HJ_1-10k_transcript/201268      AXS GO:0048040: UDP-glucuronate decarboxylase activity (molecular_function); GO:0050662: coenzyme binding (molecular_function); K12449|AXS AMM04380.1 1.0000000000000001E-217 359/390(92.05)   762 UDP-D-apiose/UDP-D-xylose synthase [Albuca bracteata]           XM_020421333.1       0 1026/1192(86.07)  1267                                                                                                 sp|Q9SGE0|AXS2_ARATH 2.0000000000000001E-209 342/389(87.92)   729 UDP-D-apiose/UDP-D-xylose synthase 2 OS=Arabidopsis thaliana GN=AXS2 PE=2 SV=1    TRUE
6  HJ_1-10k_transcript/273974      AXS GO:0003824: catalytic activity (molecular_function); GO:0050662: coenzyme binding (molecular_function);                     K12449|AXS ONK60118.1 2.6000000000000001E-170 285/300(95.00)   604 uncharacterized protein A4U43_C08F14510 [Asparagus officinalis] XM_020421333.1       0 802/904(88.72)    1105                                                                                                 sp|Q9SGE0|AXS2_ARATH 3.7E-165                271/299(90.64)   582 UDP-D-apiose/UDP-D-xylose synthase 2 OS=Arabidopsis thaliana GN=AXS2 PE=2 SV=1    TRUE
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────</code></pre>
</div>
</div>
<div class="quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="Fig7.PSP.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Fig7.PSP</figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
</section>
<section id="不饱和脂肪酸代谢" class="level2">
<h2 class="anchored" data-anchor-id="不饱和脂肪酸代谢">3.1.4 不饱和脂肪酸代谢</h2>
<p>我们在之前的代谢组数据中发现在成年黄精中亚油酸和α-亚麻酸含量要显著高于幼年黄精，根据KEGG数据库的注释我们发现，α-亚麻酸<code>α-linolenic acid</code>可以被<code>E3.1.1.4</code>催化由磷脂酰胆碱<code>Phosahatidylcholine</code>水解形成，亚油酸同样可以通过<code>E3.1.1.4</code>由卵磷脂水解而来，通常植物中的<code>E3.1.1.4</code>主要为<code>triacylglycerol lipase SDP1 (SDP1/TGL4)</code>, <code>Secretory phospholipase A2 (PLA2G)</code>,<code>phospholipase A1 (LCAT3)</code> ，而亚油酸和α-亚麻酸在植物中会通过<code>lipoxygenase (LOX)</code>氧化生成<code>HPODE</code>或者<code>HpOTrE</code>,本研究鉴定到78个相关isoform，其中DET仅有38个，合并后我们整体看了下他们在幼年黄精和成年黄精之间的差异，结果</p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>√ Successfully imported: 80 obs. of 20 variables</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
                       GeneID category                                                                                                                                                                                                                                                                                           GO_Anno   KEGG_entry             ID E_value       Identity Score                                                           Annotation             ID E_value         Identity Score                                                                                Annotation                   ID               E_value       Identity Score                                                                        Annotation DET set
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
3  HJ_1-10k_transcript/47099     LOX2S GO:0009536: plastid (cellular_component); GO:0016165: linoleate 13S-lipoxygenase activity (molecular_function); GO:0031408: oxylipin biosynthetic process (biological_process); GO:0046872: metal ion binding (molecular_function); GO:0055114: oxidation-reduction process (biological_process); K00454|LOX2S XP_020256936.1       0 785/914(85.89)  1617 LOW QUALITY PROTEIN: probable lipoxygenase 6 [Asparagus officinalis] XM_020401347.1       0 2256/2792(80.80)  2154 PREDICTED: Asparagus officinalis probable lipoxygenase 6 (LOC109833605), mRNA             sp|Q8H016|LOX6_ORYSJ 0                     646/892(72.42)  1331 Probable lipoxygenase 6 OS=Oryza sativa subsp. japonica GN=Os03g0179900 PE=2 SV=2   FALSE
4  HJ_1-10k_transcript/84204     LOX2S --                                                                                                                                                                                                                                                                                                K00454|LOX2S XP_038985330.1       0 627/781(80.28)  1337 putative lipoxygenase 5 [Phoenix dactylifera]                        XM_009419421.2       0 1795/2351(76.35)  1218                                                                                           sp|Q7XV13|LOX5_ORYSJ 0                     591/781(75.67)  1247 Putative lipoxygenase 5 OS=Oryza sativa subsp. japonica GN=Os04g0447100 PE=3 SV=2   TRUE 
5  HJ_1-10k_transcript/45970     LOX2S --                                                                                                                                                                                                                                                                                                K00454|LOX2S XP_038985330.1       0 698/917(76.12)  1432 putative lipoxygenase 5 [Phoenix dactylifera]                        XM_009419421.2       0 1931/2534(76.20)  1290                                                                                           sp|Q7XV13|LOX5_ORYSJ 0                     621/841(73.84)  1296 Putative lipoxygenase 5 OS=Oryza sativa subsp. japonica GN=Os04g0447100 PE=3 SV=2   TRUE 
6  HJ_1-10k_transcript/162287    LOX2S GO:0009536: plastid (cellular_component); GO:0016165: linoleate 13S-lipoxygenase activity (molecular_function); GO:0031408: oxylipin biosynthetic process (biological_process); GO:0046872: metal ion binding (molecular_function); GO:0055114: oxidation-reduction process (biological_process); K00454|LOX2S XP_020256936.1       0 525/592(88.68)  1109 LOW QUALITY PROTEIN: probable lipoxygenase 6 [Asparagus officinalis] XM_010939026.3       0 1229/1606(76.53)  861  PREDICTED: Elaeis guineensis probable lipoxygenase 6 (LOC105056718), mRNA                 sp|Q8H016|LOX6_ORYSJ 5.69999999999999E-272 445/591(75.30)  937  Probable lipoxygenase 6 OS=Oryza sativa subsp. japonica GN=Os03g0179900 PE=2 SV=2   TRUE 
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────</code></pre>
</div>
</div>
<div class="quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="Fig8.UnFA.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Fig7.PSP</figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>

</section>
</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-fengTranscriptomeAnalysisDifferent2022" class="csl-entry" role="doc-biblioentry">
Feng, Tinghui, Yujie Jiang, Qiaojun Jia, Ruilian Han, Dekai Wang, Xuemin Zhang, and Zongsuo Liang. 2022. <span>“Transcriptome <span>Analysis</span> of <span>Different Sections</span> of <span>Rhizome</span> in <span>Polygonatum</span> Sibiricum <span>Red</span>. And <span>Mining Putative Genes Participate</span> in <span>Polysaccharide Biosynthesis</span>.”</span> <em>Biochemical Genetics</em> 60 (5): 1547–66. <a href="https://doi.org/10.1007/s10528-022-10183-x">https://doi.org/10.1007/s10528-022-10183-x</a>.
</div>
<div id="ref-wangNovoAssemblyAnalysis2017" class="csl-entry" role="doc-biblioentry">
Wang, Shiqiang, Bin Wang, Wenping Hua, Junfeng Niu, Kaikai Dang, Yi Qiang, and Zhezhi Wang. 2017. <span>“De <span>Novo Assembly</span> and <span>Analysis</span> of <span>Polygonatum</span> Sibiricum <span>Transcriptome</span> and <span>Identification</span> of <span>Genes Involved</span> in <span>Polysaccharide Biosynthesis</span>.”</span> <em>International Journal of Molecular Sciences</em> 18 (9): 1950. <a href="https://doi.org/10.3390/ijms18091950">https://doi.org/10.3390/ijms18091950</a>.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->



</body></html>