<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.0.36">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Shawn Wang">
<meta name="dcterms.date" content="2022-10-02">

<title>转录组数据</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
</style>


<script src="03.2代转录组分析_files/libs/clipboard/clipboard.min.js"></script>
<script src="03.2代转录组分析_files/libs/quarto-html/quarto.js"></script>
<script src="03.2代转录组分析_files/libs/quarto-html/popper.min.js"></script>
<script src="03.2代转录组分析_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="03.2代转录组分析_files/libs/quarto-html/anchor.min.js"></script>
<link href="03.2代转录组分析_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="03.2代转录组分析_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="03.2代转录组分析_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="03.2代转录组分析_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="03.2代转录组分析_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">转录组数据</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Shawn Wang </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">October 2, 2022</p>
    </div>
  </div>
    
  </div>
  

</header>

<center>
<p><font color="salmon" size="2pt"> 🏡Zhang Lab, Aug 05, 2022 🏡</font></p>
<p><font color="green" size="2pt"> 🔬Multi-omics research center, State Key Laboratory of Crop Stress Adaption and Improvement, HENU, Kaifeng, Henan 🔬</font></p>
</center>
<section id="ngs转录组分析" class="level1">
<h1>2 NGS转录组分析</h1>
<section id="方法" class="level2">
<h2 class="anchored" data-anchor-id="方法">2.1 方法</h2>
<ol type="1">
<li><p>以三代全长转录本作为参考序列，将每个样品的clean reads（二代数据）比对回参考序列。该过程我们采用了<code>RSEM</code>软件，<code>RSEM</code>流程中选择使用<code>bowtie2</code>软件，参数为<code>bowtie2</code>默认参数. RSEM对bowtie的比对结果进行统计，进一步得到了每个样品比对到每个基因上的readcount 数目，并对其进行FPKM转换，进而分析基因的表达水平。</p></li>
<li><p>材料间的相关性使用斯皮尔曼相关计算，PCA结果通过PCAtools包完成。</p></li>
<li><p>差异比对软件使用的是<code>DESeq2</code>。</p></li>
<li><p>使用所有差异表达的基因作为基因集合，然后通过<code>mFuzz</code>对这些基因进行分类，我们选择前期和后期差异明显的cluster进行后续分析。</p></li>
<li><p>富集分析使用通过TBtools完成。</p></li>
</ol>
</section>
<section id="结果" class="level2">
<h2 class="anchored" data-anchor-id="结果">3.1 结果</h2>
<section id="global-transcriptomic-changes-of-rhizome-of-p.-sibniricum-between-juvenal-and-adult" class="level3">
<h3 class="anchored" data-anchor-id="global-transcriptomic-changes-of-rhizome-of-p.-sibniricum-between-juvenal-and-adult">3.1.1 Global transcriptomic changes of rhizome of <em>P. sibniricum</em> between juvenal and adult</h3>
<p>为了搞清楚幼年黄精和成年黄精块茎在药效方面的差异，我们以3代全长转录组检测到的转录本为参考，将2代测序的clean reads比对回参考序列，对不同时期的野外黄精块茎样本进行精确的定性和定量。 结果表明平均每个样品产生了78.25 Mb的clean reads, 平均覆盖度为91.55% (Table S9)样品间的相关性表明分属juvenal组的样品和adult组的样品之间相关性较低。由于野外取样的原因，个体差异较为明显，生物学重复间的差异较大，特别是幼年材料，按大小划分整体不容易区分，对于成年材料来说，对于成年材料来说，size5和size6的生物学重复较好，但两者间的相关性非常低，说明差异较大（Figure5 A）,同时，我们对log2转换后的FPKM值计算了欧式距离，并进行了层级聚类，距离方法为<code>complete</code>,同时也做了PCA分析，结果显示不同年份的材料较好的聚集在一起(Figure S5 A,B)，且Size5和其他材料差异较大。 为了研究不同年份黄精之间的差异，我们对他们做了成对的差异分析，与材料相关性分析结果一致，size5和size6与其他组合差异基因数量最多，同时size5和size6之间也存在较大差异。综上，我们发现在转录水平幼年黄精和成年黄精间确实存在明显差异，而两个成年黄精材料间也存在较为明显的差异，尽管幼年黄精间可能由于个体差异的关系生物学重复不太理想，造成重复间相关性较低，但是在压缩数据变异度后（对FPKM取log）不同大小的黄精块茎材料间的差异还是被凸显出来，所以通过经验判断黄精年份的方法确实存在一定问题，但是在转录水平确实体现出一定的趋势，同时在如此大的个体差异存在的情况下，幼年黄精和成年黄精间显著的差异表明黄精在生长发育到一定阶段后，块茎内正在进行的生物学过程及代谢途径确实发生了显著的改变。从幼年黄精和成年黄精材料的转录及代谢数据中去规避个体差异造成的影响从而挖掘真实的年份积累造成的差异成为本研究关注的重点。</p>
<div id="fig5.DEG_overview" class="quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div id="Def" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Fig5.DEG_overview.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Figure5.DEG.overview</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="FigS5.NGS_Trans_Overview.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">FigureS5.DEG.overview2</figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
</section>
</section>
<section id="time-course-transcriptome-analysis" class="level2">
<h2 class="anchored" data-anchor-id="time-course-transcriptome-analysis">3.1.2 Time course transcriptome analysis</h2>
<p>为了更好的了解这些DEGs在不同年份之间的表达变化，我们通过mFuzz对DEGs做了时序分析，这些DEGs被分成了12个cluster。这些DETs表现出多样的趋势，在这些expression pattern中，我们更加关注在Adult中特异表达的DEGs，其中cluster2,8,9表现出明显的在材料size5中高表达，而在其他时期相对较低（Figure 5 A）。通过后续的GO富集分析我们发现，cluster2的DEGs更多的影响的是非编码RNA代谢通路<code>ncRNA metabolic process</code>,泛素依赖性蛋白质分解代谢过程<code>ubiquitin−dependent protein catabolic process</code>(Figure S6 A)。cluster8与cluster2的功能类似，同样也是影响 <code>RNA metabolic process</code>，<code>protein modification process</code>.然而 cluster9则显著富集在碳水化合物代谢，多糖代谢过程等通路（Figure 6 B）从表达模式上看，Size5中，除了在nuclear metabolic和protein modification这些pathway上调之外，多糖代谢，特别是多糖合成代谢也上调了。而同属Adult的size6特意表达的基因在cluster11，GO富集分析的结果显示这些DEGs更多的参与到了氧化还原，有机酸的生物合成代谢通路、苯丙烷的代谢通路中（Figure 6 C）从cellular component trem的富集结果看，这些DEGs与质膜相关，参与了植物细胞壁的形成。此外我们还发现在幼年黄精材料中特异性表达的一些cluster，比如cluster3在Size3阶段特意高表达，cluster4在size4阶段特意高表达，cluster10在size2阶段高表达，cluster12在size1阶段高表达，cluster5在size1-2阶段高表达。综上，我们通过时序分析挖掘出老年黄精和幼年黄精在整体表达模式上的异同，结果表明，相对于幼年黄精，成年黄精在<code>RNA metabolic process</code>，<code>protein modification process</code>，<code>obsolete oxidation-reduction progress</code>,<code>phenylpropanoid metabolic process</code>较为活跃，此外在Size5的材料中我们发现了成年黄精在多糖合成代谢中要明显高于幼年黄精，而且在Size6中质膜，植物细胞壁合成的方面也要高于幼年黄精。</p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>√ Successfully imported: 51 obs. of 6 variables</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
                      Gene_ID        KEGG_entry                  KOG_class_annotation                                                                                      Uniprot Annotation                                                                                                                                     NR Annotation                                                                                                   NT Annotation
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
1  HJ_1-10k_transcript/267397 K00700|GBE1, glgB --                                    1,4-alpha-glucan-branching enzyme 1, chloroplastic/amyloplastic-like [Asparagus officinalis]            PREDICTED: Asparagus officinalis 1,4-alpha-glucan-branching enzyme 1, chloroplastic/amyloplastic-like (LOC109821070), mRNA                        1,4-alpha-glucan-branching enzyme, chloroplastic/amyloplastic OS=Oryza sativa subsp. japonica GN=SBE1 PE=1 SV=2
2  HJ_1-10k_transcript/61446  K00700|GBE1, glgB Carbohydrate transport and metabolism 1,4-alpha-glucan-branching enzyme 1, chloroplastic/amyloplastic-like [Asparagus officinalis]            PREDICTED: Asparagus officinalis 1,4-alpha-glucan-branching enzyme 1, chloroplastic/amyloplastic-like (LOC109821070), mRNA                        1,4-alpha-glucan-branching enzyme, chloroplastic/amyloplastic OS=Oryza sativa subsp. japonica GN=SBE1 PE=1 SV=2
3  HJ_1-10k_transcript/162055 K00688|PYG, glgP  --                                    alpha-1,4 glucan phosphorylase L isozyme, chloroplastic/amyloplastic isoform X2 [Asparagus officinalis]                                                                                                                                                   Alpha-1,4 glucan phosphorylase L-2 isozyme, chloroplastic/amyloplastic OS=Solanum tuberosum GN=STP-1 PE=1 SV=1 
4  HJ_1-10k_transcript/310449 K00688|PYG, glgP  --                                    alpha-1,4 glucan phosphorylase L isozyme, chloroplastic/amyloplastic-like isoform X1 [Glycine soja]                                                                                                                                                       Alpha-1,4 glucan phosphorylase L-2 isozyme, chloroplastic/amyloplastic OS=Solanum tuberosum GN=STP-1 PE=1 SV=1 
5  HJ_1-10k_transcript/14006  K01209|abfA       --                                    alpha-L-arabinofuranosidase 1-like [Asparagus officinalis]                                              PREDICTED: Asparagus officinalis alpha-L-arabinofuranosidase 1-like (LOC109841490), mRNA                                                          Alpha-L-arabinofuranosidase 1 OS=Arabidopsis thaliana GN=ASD1 PE=1 SV=1                                        
6  HJ_1-10k_transcript/295959 K01177|E3.2.1.2   --                                    Beta-amylase 2, chloroplastic [Cocos nucifera]                                                          --                                                                                                                                                Beta-amylase 1, chloroplastic OS=Arabidopsis thaliana GN=BAM1 PE=1 SV=1                                        
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────</code></pre>
</div>
</div>
<div id="fig6.mFuzz" class="quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<p><img src="Fig6.mFuzz.png" class="img-fluid" alt="Figure6.mFuzz"> <img src="cluster9.heatmap.png" class="img-fluid" alt="Figuretmp1.heatmap"></p>
</div>
</div>
</div>
</section>
<section id="黄精多糖合成代谢通路分析" class="level2">
<h2 class="anchored" data-anchor-id="黄精多糖合成代谢通路分析">3.1.3 黄精多糖合成代谢通路分析。</h2>
<div id="fig6.mFuzz" class="quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="iShot_2022-10-08_09.41.52.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Figure_ref</figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->



</body></html>